playbooks/gce/openshift-node/terminate.yml

```diff
@@ -16,7 +16,7 @@
   hosts: localhost
   connection: local
   tasks:
-    - name: Terminate minion instances
+    - name: Terminate node instances
```

Comment:
Just an fyi, since you changed the name from openshift-minion to openshift-node, all existing clusters will _not_ be able to be terminated with this playbook. The existing instances have a host-type of openshift-minion. This terminate will be looking for instances with a host-type of openshift-node (passed in as host_group_exp).

We may want to leave around the openshift-minion directory and previous terminate.yml file in it for backwards compat. But that's the only file I'd leave behind.